e2e/azure-node-e2e/tests/azure-node.spec.ts

```diff
@@ -43,4 +44,40 @@ describe('azure-node e2e', () => {
             expect(fileContent).toMatch(/appInsights\.start/g);
         }, 120000);
     });
+
+    describe('app-insights-deployment generator', () => {
+        const project = uniq('nextjs');
+
+        beforeAll(async () => {
+            // Generate next project
+            await runNxCommandAsync(
+                `generate @nx/next:application ${project} --custom-server --e2eTestRunner=none --no-appDir`,
+            );
+            // Run next init generator
+            await runNxCommandAsync(
+                `generate @ensono-stacks/next:init --project=${project} --no-interactive`,
+            );
+            // Run next deployment generator
+            await runNxCommandAsync(
+                `generate @ensono-stacks/next:init-deployment --project=${project} --no-interactive`,
+            );
+        });
+
+        it('runs the install generator', async () => {
```

Comment:
Great stuff! Much more comprehensive now that we're actually fully testing the generator and it's integration with dependencies.